app/core/utils.coffee

```diff
@@ -215,12 +215,21 @@ if document?.createElement
 module.exports.userAgent = ->
   window.navigator.userAgent
 
-module.exports.getQueryVariable = getQueryVariable = (param, defaultValue) ->
-  query = document.location.search.substring 1
+exports.getDocumentSearchString = ->
+  # moved to a separate function so it can be mocked for testing
+  return document.location.search
+
+module.exports.getQueryVariables = ->
+  query = exports.getDocumentSearchString().substring(1)
```

Comment:
Yep, it works, figured it was shorter. This has blossomed into a larger discussion of the best style for exporting. For this PR, I'll just have it use the same style as the rest of the file.